tests/AssertTest.php

```diff
@@ -289,8 +290,18 @@ public function getTests()
             array('uuid', array('ff6f8cb0-c57da-51e1-9b21-0800200c9a66'), false),
             array('uuid', array('af6f8cb-c57d-11e1-9b21-0800200c9a66'), false),
             array('uuid', array('3f6f8cb0-c57d-11e1-9b21-0800200c9a6'), false),
-
+            array('throws', array(function() { throw new LogicException('test'); }, 'LogicException'), true),
+            array('throws', array(function() { throw new LogicException('test'); }, 'IllogicException'), false),
+            array('throws', array(function() { throw new Exception('test'); }), true),
         );
+        if (version_compare(PHP_VERSION, '7.0.0') >= 0) {
+            $tests = array_merge($tests, array(
+                array('throws', array(function() { trigger_error('test'); }, 'Throwable'), true),
+                array('throws', array(function() { trigger_error('test'); }, 'Unthrowable'), false),
```

Comment:
What is the difference?

What other test case does `throw new Error()` cover that is not already covered? Or is it just a matter of taste?